
class UrLContentError(Exception):
    """ Raised when the desired content is not found within url """
    def __init__(self, *args, **kwargs):
        pass
